resources/dicts/events/misc_events/general/apprentice.json

```diff
@@ -129,5 +129,23 @@
     "other_cat_trait": null,
     "other_cat_skill": null,
     "accessories": ["COLLAR"]
-  }
+  },
+  {
+    "camp": "any",
+    "tags": ["other_cat", "other_cat_app", "Newleaf", "Greenleaf", "Leaf-fall", "to_both", "platonic", "trust", "comfort", "neg_dislike"],
+    "event_text": "m_c and r_c practice climbing when they found a nest of strange eggs.",
```

Comment:
grammar is a bit weird here, the tenses are flipping in the middle. changing 'practice' to 'were practicing' would fix it.